providers/flagd/README.md

```diff
@@ -116,6 +116,7 @@ Given below are the supported configurations:
 | streamDeadlineMs      | FLAGD_STREAM_DEADLINE_MS       | int                      | 600000    | rpc & in-process        |
 | keepAliveTime         | FLAGD_KEEP_ALIVE_TIME_MS       | long                     | 0         | rpc & in-process        |
 | selector              | FLAGD_SOURCE_SELECTOR          | String                   | null      | in-process              |
+| providerID            | FLAGD_SOURCE_PROVIDER_ID       | String                   | null      | in-process              |
```

Comment:
```suggestion
| providerId            | FLAGD_SOURCE_PROVIDER_ID       | String                   | null      | in-process              |
```

I think for consistency this should be fully camel case.

Reply:
Haha, I think I'm getting conflicting feedback here :) I started with camelcase and then @beeme1mr suggested capitalized ID (to be consistent with Go) and now you are suggesting camelcase again. Personally, I don't have a strong opinion either way.

Reply:
Sorry! https://github.com/open-feature/java-sdk-contrib/pull/1259#discussion_r1981520856